knowledge-base/how-to-view-web-session-summary.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to view Web Session Summary
```

Comment:
"view" should start with a capital letter to be consistent with the titles of all other KB articles